### PR TITLE
Fix #769 : Lowfi Landscape: AppVersionActivity

### LINKED ISSUE #769: Lowfi Landscape: AppVersionActivity
Mock Link: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/7ce387fc-c8c5-4993-8888-2e3b9bc64b52/RG-MN-Register-Empty-31

Add test cases for orientation mode also.

### PR DESCRIPTION
**Pull request by chrk2205**
## Explanation
Fixes #769

created landscape version of App-Version-Activity
modified android manifest accordingly

Mock Link: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/7ce387fc-c8c5-4993-8888-2e3b9bc64b52/RG-MN-Register-Empty-31

Screenshot
aaa

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by rt4914 — CHANGES_REQUESTED** [review]
Please add test case for landscape mode, you will need to add them in AppVersionActivityTest. For reference you can check landscape mode related test cases in `HelpFragmentTest`, `ConceptCardFragmentTest`

File: app/src/sharedTest/java/org/oppia/app/administratorcontrols/AppVersionActivityTest.kt [hunk 1]
```diff
@@ -62,8 +63,9 @@ class AppVersionActivityTest {
   }
 
   @Test
-  fun testAppVersionActivity_loadFragment_displaysAppVersion() {
+  fun testAppVersionActivity_configurationChanged_loadFragment_displaysAppVersion() {
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/administratorcontrols/AppVersionActivityTest.kt]: This is correct you you actually need previous test case also. Currently you have just modified the previous test case, which is incorrect. We need previous test case along with this new test case.

Also, the name should like this:
`testAppVersionActivity_configurationChange_displaysAppVersion`

**Review by rt4914 — CHANGES_REQUESTED** [review]
Changes in test case.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Actually you have written everything correctly but still there is some confusion in understanding.

Basically without changing any previous test case you just had to add this test case which you had written earlier:
```
@Test
  fun testAppVersionActivity_changeConfiguration_displaysAppVersion() {
    launchAppVersionActivityIntent().use {
      onView(isRoot()).perform(orientationLandscape())
      onView(
        withText(
          String.format(
            context.resources.getString(R.string.app_version_name),
            BuildConfig.VERSION_NAME
          )
        )
      ).check(matches(isDisplayed()))
      onView(withText(String.format(context.resources.getString(R.string.app_last_update_date), lastUpdateDate))).check(
        matches(isDisplayed())
      )
    }
  }
```

File: app/src/sharedTest/java/org/oppia/app/administratorcontrols/AppVersionActivityTest.kt [hunk 2]
```diff
@@ -78,6 +79,34 @@ class AppVersionActivityTest {
     }
   }
 
+  @Test
+  fun testAppVersionActivity_configurationChange_AppVersionIsDisplayedCorrectly(){
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/administratorcontrols/AppVersionActivityTest.kt]: first character after `_` should be small

`testAppVersionActivity_configurationChange_appVersionIsDisplayedCorrectly`

File: app/src/sharedTest/java/org/oppia/app/administratorcontrols/AppVersionActivityTest.kt [hunk 3]
```diff
@@ -78,6 +79,34 @@ class AppVersionActivityTest {
...
+        R.id.app_last_update_date_text_view
+      )
+    ).check(
+      matches(
+        withText(
+          String.format(context.resources.getString(R.string.app_last_update_date), lastUpdateDate)
+        )
+      )
+    )
+
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/administratorcontrols/AppVersionActivityTest.kt]: Remove this extra blank line

**Review by rt4914 — CHANGES_REQUESTED** [review]
PTAL nit changes.

Also check this: https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section

File: app/src/sharedTest/java/org/oppia/app/administratorcontrols/AppVersionActivityTest.kt (hunk 3 above)
Comment (chrk2205) [reply app/src/sharedTest/java/org/oppia/app/administratorcontrols/AppVersionActivityTest.kt]: done

File: app/src/sharedTest/java/org/oppia/app/administratorcontrols/AppVersionActivityTest.kt (hunk 2 above)
Comment (chrk2205) [reply app/src/sharedTest/java/org/oppia/app/administratorcontrols/AppVersionActivityTest.kt]: done

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**